state_management/javascript/http/order-processor/index.js

```diff
@@ -1,42 +1,40 @@
-import axios from "axios";
+import axios from "axios"
 
-const DAPR_HOST = process.env.DAPR_HOST || "http://localhost";
-const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3500";
-const DAPR_STATE_STORE = 'statestore'
+const DAPR_HOST = process.env.DAPR_HOST ?? "http://localhost"
+const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT ?? "3500"
+const DAPR_STATE_STORE_NAME = 'statestore'
+const stateStoreBaseUrl = `${DAPR_HOST}:${DAPR_HTTP_PORT}/v1.0/state/${DAPR_STATE_STORE_NAME}`
 
 async function main() {
-  for(var i = 1; i <= 10; i++) {
-    const orderId = i;
-    const order = {orderId: orderId}; 
-    const state = [{
-      key: orderId.toString(),
-      value: order
-    }];
+  // For each loop, Save order, Get order, and Delete order
+  for (var i = 1; i <= 10; i++) {
+    const orderId = i.toString()
+    const order = { orderId }
+    const state = [
+      {
+        key: order.orderId.toString(),
```

Comment:
why not use `orderId` here?

Reply:
It's minor, but it is more robust to use the `orderId` of the order object so it is guruanteed to be the id of the object and not some other ID.  Even though we know `orderId` was used to set the id of the order, this is not guaranteed.

Someone could technically create the order with id then change order id.

Example:
```
let orderId = "1",
const order = { orderId }
orderId = "2"

const state = [
    {
        key: orderId,
        value: order
    }
]
```

This would cause an order which has id "1" to have key "2". The code should make this not possible. Which is why we used the orderId from the order.